provider/resources.go

```diff
@@ -70,26 +60,48 @@ func Provider() tfbridge.ProviderInfo {
 		// category/cloud tag helps with categorizing the package in the Pulumi Registry.
 		// For all available categories, see `Keywords` in
 		// https://www.pulumi.com/docs/guides/pulumi-packages/schema/#package.
-		Keywords:             []string{"pulumi", "databricks", "category/Infrastructure"},
-		License:              "Apache-2.0",
-		Homepage:             "https://www.pulumi.com",
-		Repository:           "https://github.com/pulumi/pulumi-databricks",
-		GitHubOrg:            "databrickslabs",
-		Config:               map[string]*tfbridge.SchemaInfo{},
-		PreConfigureCallback: preConfigureCallback,
+		Keywords:   []string{"pulumi", "databricks", "category/Infrastructure"},
+		License:    "Apache-2.0",
+		Homepage:   "https://www.pulumi.com",
+		Repository: "https://github.com/pulumi/pulumi-databricks",
+		GitHubOrg:  "databrickslabs",
+		Config:     map[string]*tfbridge.SchemaInfo{},
+		IgnoreMappings: []string{
+			"databricks_group_member", // causes the schema to have no deterministic generation
+		},
 		Resources: map[string]*tfbridge.ResourceInfo{
-			"databricks_aws_s3_mount":          {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AwsS3Mount")},
-			"databricks_azure_adls_gen1_mount": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AzureAdlsGen1Mount")},
-			"databricks_azure_adls_gen2_mount": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AzureAdlsGen2Mount")},
-			"databricks_azure_blob_mount":      {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AzureBlobMount")},
-			"databricks_catalog":               {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Catalog")},
-			"databricks_cluster":               {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Cluster")},
-			"databricks_cluster_policy":        {Tok: tfbridge.MakeResource(mainPkg, mainMod, "ClusterPolicy")},
-			"databricks_dbfs_file":             {Tok: tfbridge.MakeResource(mainPkg, mainMod, "DbfsFile")},
-			"databricks_directory":             {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Directory")},
-			"databricks_external_location":     {Tok: tfbridge.MakeResource(mainPkg, mainMod, "ExternalLocation")},
-			"databricks_git_credential":        {Tok: tfbridge.MakeResource(mainPkg, mainMod, "GitCredential")},
-			"databricks_global_init_script":    {Tok: tfbridge.MakeResource(mainPkg, mainMod, "GlobalInitScript")},
+			"databricks_aws_s3_mount": {
+				Tok: tfbridge.MakeResource(mainPkg, mainMod, "AwsS3Mount"),
+				Docs: &tfbridge.DocInfo{
+					Markdown: []byte(" "),
+				},
+			},
+			"databricks_azure_adls_gen1_mount": {
+				Tok: tfbridge.MakeResource(mainPkg, mainMod, "AzureAdlsGen1Mount"),
+				Docs: &tfbridge.DocInfo{
```

Comment:
Suggest creating the following when there's a lot of missing docs:

```go
func noUpstreamDocs() &tfbridge.DocInfo {
 // return single whitespace thingy
}
```

Reply:
you can remove this resource :)

Reply:
Covered by https://github.com/pulumi/pulumi-databricks/issues/10